app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comment_counter` should be marked as _private_ after implementing the `after_save` method.

Reply:
@rloterh Thank you for the best review you gave me. I have followed all the required changes plus optional suggestions. Thank you very much.